lambdas/answer/src/test/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandlerTest.java

```diff
@@ -162,7 +162,7 @@ void shouldReturn200WithFinalResponseFromExperianAPI() throws IOException, SqsEx
         verify(mockSessionService).createAuthorizationCode(mockSessionItem);
         verify(mockAuditService)
                 .sendAuditEvent(
-                        eq(AuditEventType.THIRD_PARTY_REQUEST_ENDED),
```

Comment:
Can we change the test to also include `RESPONSE_RECEIVED`? 

When we remove the `THIRD_PARTY_REQUEST_ENDED` event we can then remove the associated tests.